controller/handler/patient.js

```diff
@@ -81,14 +81,17 @@ function patientView (request, reply) {
                 patient: processPatient(currentPatient),
                 trial: currentTrial,
                 surveys: surveyInstances.map((surveyInstance) => {
-                    surveyInstance.startTime = moment(surveyInstance.startTime).format('MM-DD-YYYY');
-                    surveyInstance.endTime = moment(surveyInstance.endTime).format('MM-DD-YYYY');
+                    surveyInstance.startTime = moment(new Date(surveyInstance.startTime)
```

Comment:
No no... I convert it to "new Date().ISOString()" format.
We can see a warning related to this in the console that simple strings will not be supported in Moment in future. I thought making that change right now will make our lives easy later :)